### DISCUSSION TITLE
M1编译报错。。。求救！！

### DISCUSSION THREAD

**Discussion by CN1Smile** [BoomingTech/Piccolo#288] [Q&A] [answered]
0.0.7 在M1的mac上编译报错，网上找了很多资料都没有解决，求大佬指点！！救救孩子！！
报错信息（前半部分）：
ld: warning: ignoring file /Users/zhangzhitong/Code/Piccolo/engine/source/meta_parser/3rd_party/LLVM/bin/macOS/libclang.dylib, building for macOS-arm64 but attempting to link with file built for macOS-x86_64
Undefined symbols for architecture arm64:
  "_clang_Cursor_getSpellingNameRange", referenced from:
      Cursor::getSourceFile() const in cursor.o
  "_clang_createIndex", referenced from:
      MetaParser::parse() in parser.o
  "_clang_createTranslationUnitFromSourceFile", referenced from:
      MetaParser::parse() in parser.o
  "_clang_disposeIndex", referenced from:
      MetaParser::~MetaParser() in parser.o
  "_clang_disposeString", referenced from:
      Utils::toString(CXString const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >&) in meta_utils.o
  "_clang_disposeTranslationUnit", referenced from:
      MetaParser::~MetaParser() in parser.o
  "_clang_getArgType", referenced from:
      CursorType::GetArgument(unsigned int) const in cursor_type.o

**Comment by Ol6rin** [comment, marked as answer]
感谢反馈！目前有一条相关的PR #284 正在测试中，如果通过我们会尽快合并。
